src/game/Params.ts

```diff
@@ -0,0 +1,42 @@
+export type Params = {
+    vars: {
+        [id: string]: number
+    }
+    flags: {
+        [id: string]: boolean
+    }
+}
+
+export type ParamQuery = {
```

Comment:
Another good idea to separate distinct concepts such as `Params` to make them easier to reason about.